pkg/sippyserver/server.go

```diff
@@ -97,10 +97,9 @@ func (s *Server) refresh(w http.ResponseWriter, req *http.Request) {
 
 func (s *Server) RefreshData() {
 	klog.Infof("Refreshing data")
-	s.bugCache.Clear()
```

Comment:
Is removing the bug cache clearance intentional?

Reply:
It was yes, as far as I can see, there is no bug cache in play anymore, the data is included in the reports file. I will look into removing s.bugCache and see if that surfaces anything I missed on Monday.

Reply:
I did find it used here: https://github.com/openshift/sippy/blob/38ca4dec6de0c2b5db3a20c2d15a16cc8fb42cf0/pkg/sippyserver/server.go#L219

Very strange function, seems insanely slow to be hooked up to a live web request. I'm guessing that is somewhat legacy?

In any case, is the best course of action to restore the bug cache clearing and leave it be for now?

Reply:
Updated.